Remotion/ObjectBinding/Web/UI/Controls/BocBooleanValueImplementation/Rendering/BocBooleanValueRenderer.cs

```diff
@@ -297,25 +297,25 @@ private void RegisterStarupScriptIfNeeded (BocBooleanValueRenderingContext rende
       {
         checkedState = HtmlAriaCheckedAttributeValue.Mixed;
         imageUrl = resourceSet.NullIconUrl;
-        description = string.IsNullOrEmpty (renderingContext.Control.NullDescription)
-            ? resourceSet.DefaultNullDescription
-            : renderingContext.Control.NullDescription;
+        description = renderingContext.Control.NullDescription.IsEmpty
```

Comment:
Please change type of description to WebString. That way, we don't do this in every line, removing the code duplication and syntactical noise.